deis-lite/manifests/deis-builder-rc.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-builder
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    name: deis-builder
```

Comment:
we can merge this one first, then I will make another PR to address `app` for selectors